src/twisted/python/_release.py

```diff
@@ -455,16 +456,16 @@ def _writeSection(self, fileObj, header, tickets):
             reverse[description].sort()
         reverse = reverse.items()
         # result is a tuple of (descr, tickets)
-        reverse.sort(key=lambda result: result[1][0])
+        reverse = sorted(reverse, key=lambda result: result[1][0])
 
-        fileObj.write(header + '\n' + '-' * len(header) + '\n')
+        fileObj.write((header + u'\n' + u'-' * len(header) + u'\n').encode("utf-8"))
         for (description, relatedTickets) in reverse:
             ticketList = ', '.join([
                 '#' + str(ticket) for ticket in relatedTickets])
             entry = ' - %s (%s)' % (description, ticketList)
```

Comment:
I think it would be better to use `format` method here to keep the code consistent